langserver/workspace_refs.go

```diff
@@ -67,6 +67,7 @@ func (h *LangHandler) handleWorkspaceReferences(ctx context.Context, conn JSONRP
 			}
 		}
 		unvendoredPackages[bpkg.ImportPath] = struct{}{}
+		unvendoredPackages[bpkg.ImportPath+"_test"] = struct{}{}
```

Comment:
This is roughly the same as doing `strings.TrimSuffix(pkg, "_test")` inside the other reference code.